apps/test/unit/templates/teacherDashboard/OwnedPlSectionsTableTest.js

```diff
@@ -0,0 +1,263 @@
+import {fireEvent, render, screen, within} from '@testing-library/react';
+import React from 'react';
+import {Provider} from 'react-redux';
+
+import {
+  getStore,
+  registerReducers,
+  stubRedux,
+  restoreRedux,
+} from '@cdo/apps/redux';
+import {UnconnectedOwnedPlSectionsTable as OwnedPlSectionsTable} from '@cdo/apps/templates/teacherDashboard/OwnedPlSectionsTable';
+import teacherSections, {
+  setSections,
+} from '@cdo/apps/templates/teacherDashboard/teacherSectionsRedux';
+import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
+import i18n from '@cdo/locale';
+
+import {assert, expect} from '../../../util/reconfiguredChai';
+
+const plSectionRowData = [
+  {
+    id: 1,
+    name: 'first section',
+    studentCount: 3,
+    code: 'ABC',
+    courseOfferingsAreLoaded: true,
+    grades: ['K'],
```

Comment:
This is a very small thing, but do we have grades for PL sections?

Reply:
no, actually, PL sections can [only have the grade](https://github.com/code-dot-org/code-dot-org/blob/0659952954efa4569b999231004d9117cb444455/dashboard/app/models/sections/section.rb#L121) `pl`

Reply:
Oh good point! I had just copied these over from the test file and didn't notice, I'll clean that up!